web/src/components/FeedbackToolbarItem.tsx

```diff
@@ -13,31 +13,48 @@ import ToolbarItem from './ToolbarItem'
 type FeedbackToolbarItemProps = {
   route: RouteType
   slug?: string
-  isInBottomActionSheet: boolean
 }
 
-const FeedbackToolbarItem = ({ route, slug, isInBottomActionSheet }: FeedbackToolbarItemProps): ReactElement => {
+const FeedbackToolbarItem = ({ route, slug }: FeedbackToolbarItemProps): ReactElement => {
   const { cityCode, languageCode } = useCityContentParams()
   const [isFeedbackOpen, setIsFeedbackOpen] = useState(false)
   const [isSubmitted, setIsSubmitted] = useState(false)
+  const [isPositiveRating, setIsPositiveRating] = useState<boolean | null>(null)
   const { t } = useTranslation('feedback')
   const title = isSubmitted ? t('thanksHeadline') : t('headline')
 
   return (
     <>
       {isFeedbackOpen && (
-        <Modal title={title} closeModal={() => setIsFeedbackOpen(false)} wrapInPortal={isInBottomActionSheet}>
+        <Modal title={title} closeModal={() => setIsFeedbackOpen(false)} wrapInPortal>
           <FeedbackContainer
             onClose={() => setIsFeedbackOpen(false)}
             onSubmit={() => setIsSubmitted(true)}
             routeType={route as FeedbackRouteType}
             cityCode={cityCode}
             language={languageCode}
             slug={slug}
+            isPositiveRating={isPositiveRating}
+            setIsPositiveRating={setIsPositiveRating}
           />
         </Modal>
       )}
-      <ToolbarItem icon={FeedbackIcon} text={t('feedback')} onClick={() => setIsFeedbackOpen(true)} />
+      <ToolbarItem
+        icon={HappySmileyIcon}
+        text={t('useful')}
+        onClick={() => {
+          setIsFeedbackOpen(true)
+          setIsPositiveRating(true)
+        }}
+      />
+      <ToolbarItem
+        icon={SadSmileyIcon}
+        text={t('notUseful')}
+        onClick={() => {
+          setIsFeedbackOpen(true)
+          setIsPositiveRating(false)
+        }}
+      />
```

Comment:
:question: Wouldn't it make more sense to just render the `FeedbackToolbarItem` twice? I think this way its pretty confusing that one feedback toolbar item renders two toolbar items. That way, there would also be no need to raise the `ispositiveRating` state to this compoennt

Reply:
Hmm, interesting idea, I guess we could do that? Or at least rename the component. I guess I'll open another PR for that 😅

Reply:
Sorry, I didn't get that the PR is already merged. If you feel motivated, otherwise this is alright as well.

Reply:
All good, you're right, it's a good idea :)